File: src/main/java/ru/ioleksiv/telegram/bot/core/model/objects/IUserFrom.java
package ru.ioleksiv.telegram.bot.core.model.objects;

import ru.ioleksiv.telegram.bot.api.model.objects.std.User;
import ru.ioleksiv.telegram.bot.core.model.ITelegram;

public interface IUserFrom extends ITelegram {
    User getFrom();

    void setFrom(User from);
}

File: server-api/src/main/kotlin/io/github/oleksivio/tl/kbot/server/api/model/objects/IUserFrom.kt
package io.github.oleksivio.tl.kbot.server.api.model.objects

import io.github.oleksivio.tl.kbot.server.api.model.ITelegram
import io.github.oleksivio.tl.kbot.server.api.objects.std.User

interface IUserFrom : ITelegram {
    var from: User?
}
